app/models/bank_account.rb

```diff
@@ -28,4 +28,14 @@ def import_from_file(file, options = {})
 
   def find_import_method
   end
+
+  def assign_unchecked_transactions
+    count = 0
+    bank_transactions.without_financial_link.includes(:supplier, :user).each do |t|
+      if t.assign_to_ordergroup || t.assign_to_invoice
+        count += 1
+      end
+    end
```

Comment:
I wonder if this should be wrapped in a transaction: the iteration is done outside of the update transaction.

Reply:
i see no need for transactions, since they are independent at this level. there are transactions for the financial_links